src/main/java/com/ericsson/ei/frontend/utils/BackEndInformationControllerUtils.java

```diff
@@ -156,8 +157,9 @@
                         getHeaders(), HttpStatus.BAD_REQUEST);
             }
         } catch (Exception e) {
+            LOG.error(ExceptionUtils.getStackTrace(e));
```

Comment:
It would be good to have e.getMessage... also,, so we can get the error text message in the log error message.

Reply:
the stacktrace contains the message as well so there should be no need for that :)